packages/blade/src/components/StepGroup/__tests__/StepGroup.web.test.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
Are these needed? I'm unsure about the `no-unused-vars` rule.

Reply:
ops nope. none of these are needed. I copy-pasted base test code from somewhere so it came with that 😆 Removed them all